src/qml/ProjectCreationScreen.qml

```diff
@@ -0,0 +1,346 @@
+          }
+        }
+      }
+
+      QfExpandableGroupBox {
+        id: trackPositionGroupBox
+        title: qsTr("Track your position?")
+        width: parent.width
+        checked: true
+        content: Column {
+          id: trackPositionColumn
+          anchors.left: parent.left
+          anchors.right: parent.right
+
+          Label {
+            text: qsTr("Record your location every second, along with date and time, to keep a precise track of your movements.")
+            font: Theme.defaultFont
+            color: Theme.secondaryTextColor
+            wrapMode: Text.WordWrap
+            width: parent.width
+          }
+
+          CheckBox {
+            id: autoTrackPositionCheckBox
+            text: qsTr("Begin tracking automatically")
+            font: Theme.defaultFont
+            indicator.height: 16
+            indicator.width: 16
+            indicator.implicitHeight: 24
+            indicator.implicitWidth: 24
+            leftPadding: 1
+            checked: true
+          }
+        }
+      }
+
+      QfExpandableGroupBox {
+        id: qfieldCloudGroupBox
+        title: qsTr("Backup & collaborate?")
+        width: parent.width
+        checked: true
+        icon: "qrc:/themes/qfield/nodpi/ic_cloud_active_24dp.svg"
+
+        content: Column {
+          id: databaseAndColabrationColumn
+          width: parent.width
+          spacing: 0
+
+          Label {
+            text: qsTr("QFieldCloud allows to synchronize and merge the data collected by your team in QField. From small individual projects to large data collection campaigns. ") + "<a href=\"https://qfield.cloud/\">" + qsTr("Learn more about QFieldCloud here") + "</a>."
+            font: Theme.defaultFont
+            color: Theme.secondaryTextColor
+            wrapMode: Text.WordWrap
+            width: parent.width
+            textFormat: Text.RichText
+            onLinkActivated: link => Qt.openUrlExternally(link)
+          }
+        }
+      }
+    }
+  }
+
+  QfButton {
+    id: createProjectButton
+    anchors.bottom: parent.bottom
+    anchors.left: parent.left
+    anchors.right: parent.right
+    anchors.margins: 16
+    text: qsTr("Create Project")
+
+    onClicked: {
+      let selectedBasemap = (baseMapList.currentIndex >= 0 && baseMapList.currentIndex < baseMapList.model.length) ? baseMapList.model[baseMapList.currentIndex].name : "Colorful";
+      const isCustomBasemap = (selectedBasemap === "Custom");
+      selectedBasemap = isCustomBasemap ? baseMapURL.text : selectedBasemap;
+      var projectConfig = {
+        "name": projectName.text,
+        "isCustomBasemapSelected": isCustomBasemap,
+        "basemap": selectedBasemap,
+        "takeNotes": takeNotesGroupBox.checked,
+        "takeMediaAttachments": takeMediaCheckBox.checked,
+        "trackPosition": trackPositionGroupBox.checked,
+        "autoTrackPosition": autoTrackPositionCheckBox.checked,
+        "useCloud": qfieldCloudGroupBox.checked
```

Comment:
Please align the keys with that: https://github.com/opengisch/QField/pull/6652

For cloud, name is "use_cloud"